docs/Contribute.md

```diff
@@ -41,7 +41,7 @@ If **(and only if)** the bug hasn't been reported yet, you can [open an issue](h
 
 ## How to create a Pull Request
 
-- Read [How to create a PR](How-to-create-a-PR).
+- Read [How to create a PR](how-to-create-a-pr).
```

Comment:
we need a way to make old links work too, otherwise this is counter-productive. We've been linking pages like that everywhere

Reply:
all files in the wiki that used the old I have changed to the new

Reply:
@Kitzunu yes, I understand that. But we need to also make sure the old links work because they are linked in other pages in github and around the web.

Reply:
Maybe now it is ok

Reply:
I added

---redirect_from: "/How-to-make-a-PR"---

so it should work

Reply:
Made https://github.com/azerothcore/wiki/pull/508 to test

Reply:
works